features/support/ui/pages/money_manager.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../page'
+
+module UI::Pages
+  class MoneyManager < UI::Page
+    set_url '/en/tools/money-manager/'
+
+    element :received_first_payment_true,
+            '#questionnaire_received_first_payment_true'
```

Comment:
@margOnline @willhall88 @tomas-stefano what do you think about changing the standards on the naming convention for identifying models for the page object elements. 

The idea is to add `.t-` to help identify classes that are used for test purposes rather than styling. 

My question is, is it okay to drop this _standard_ given that Rails automatically creates unique ids for some elements?

Please give your suggestions so that we can get this out of the door.